package.json

```diff
@@ -50,15 +50,16 @@
     "preq": "^0.5.6",
     "service-runner": "^2.7.7",
     "swagger-router": "^0.7.1",
-    "swagger-ui": "git+https://github.com/wikimedia/swagger-ui#master"
+    "swagger-ui": "git+https://github.com/wikimedia/swagger-ui#master",
```

Comment:
unrelated: you should probably update to a newer service-template-node, we've retired this fork long time ago.